src/django/api/migrations/0073_add_extended_profile_switch.py

```diff
@@ -0,0 +1,16 @@
+from django.db import migrations
+
+
+def create_embedded_map_switch(apps, schema_editor):
```

Comment:
To avoid confusion I would suggest renaming this function `create_extended_profile_switch`

Reply:
Good catch!